website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: tsp-client usage
+title: Supported commands
 ---
 
```

Comment:
also wondering if you should hjave the install step as a separate page as you need it for both docs

Reply:
In my experience folks dont like to really read through multiple pages of docs, so I'd rather they have all the steps they need on the same page. We can look at this again in the future if there are more shared instructions between both pages.

Reply:
Sorry I thought this comment was on the quickstart page...in my comment below I agree to splitting up this doc into separate sections